FV3GFS/FV3GFS_io.F90

```diff
@@ -5888,31 +6752,111 @@ subroutine gfdl_diag_output(Time, Atm_block, IPD_Data, nx, ny, fprint, &
             endif
 
            !!!! Accumulated diagnostics --- lmh 19 sep 17
-           if (fprint .or. prt_stats) then
+           if (fprint .and. prt_stats) then
            select case (trim(Diag(idx)%name))
-           case('totprcp_ave')
+           case('totprcpb_ave')
```

Comment:
This is a potentially problematic name change, that needs to be pointed out in the release notes.

Reply:
I will add the following to the release notes if you agree:
"Diagnostic totprcp_ave has been renamed to "totprcpb_ave"